apps/innovations/v1-innovation-support-create/validation.schemas.ts

```diff
@@ -28,7 +28,7 @@ export const BodySchema = Joi.object<BodyType>({
   message: Joi.string().max(TEXTAREA_LENGTH_LIMIT.xxl).trim().required(),
   file: InnovationFileSchema,
   accessors: Joi.when('status', {
-    is: InnovationSupportStatusEnum.ENGAGING,
+    is: Joi.string().valid(InnovationSupportStatusEnum.ENGAGING, InnovationSupportStatusEnum.WAITING),
```

Comment:
I think in here you don't need the JOI validation since you are already validating it above:

```
is: [InnovationSupportStatusEnum.ENGAGING, InnovationSupportStatusEnum.WAITING]
```